vnext/proposals/HandleLightDarkTheme.md

```diff
@@ -0,0 +1,166 @@
+- Having access to a theme changed event allows a React Native developer to better brand and customize their app for Windows, by accommodating their own styles based on the theme selected by the user *(typically set through system settings)*
+- Exposing the Windows theme up to the JavaScript layer means React Native apps can blend more seamlessly into the Fluent Design system and native Windows platform more easily
+- Both [Android](https://developer.android.com/preview/features/darktheme) and [iOS 13](https://developer.apple.com/documentation/appkit/supporting_dark_mode_in_your_interface) have or will have support for Dark themes, thus it is a necessity that we expose some way of detecting and handling theme changes at the RN layer.
+
+### Scope
+There are two core aspects that need to be available to the app developer in order to have a Windows Light/Dark theme sensitive app:
+
+|# |Feature |
+|:-|:-|
+| 1 | Expose a Windows ThemeChanged API event at the JavaScript layer |
+| 2 | Ship a separate Windows Native Module that enables a React Native developer to access our theme brushes |
+
+- The first feature enables a React Native developer to be conscious of the Windows theme change events, and alter their style(s) accordingly if desired
+- The second feature expands the theme customization further, by allowing an app-developer at the React Native layer to access our Windows-defined theme brushes in their styles
+
+## Basic examples
+
+### Example 1 : Set up and behaviors on theme changed
+In this example, we'll look at three things:
+
+- How to set up your React Native app to be style and event sensitive to the system themes
+- How to switch styles when a theme change has occurred
+- Handling a theme changed event
+
+#### Setting up your app to be sensitive to theme changes
+First import the Platform API into your React Native app.
+
+```JavaScript
+import { AppTheme } from 'react-native-windows'
+```
+
+Create a local variable to use in style conditionals or to reference elsewhere easily.
+
+```JavaScript
+class MyAppClass extends Component {
+  state = {
+    appThemeState: AppTheme.currentTheme,
+  };
+  ...
+}
+```
+
+#### Switching styles based on the app's theme
+If the app author wants to switch the style of their component manually based on the system's theme (Dark or Light), they can do so with CSS style conditionals.
+
+```JavaScript
+<TextInput
+    style={[styles.inputStyle, this.state.appThemeState == AppTheme.dark ? styles.darkInput : styles.lightInput]}
+</TextInput>
+```
+
+#### Handling a theme changed event
+In this case an app author would like to do something or preform some behavior when the app's theme has changed.
+
+> Note: AppThemeState will be a subclass of **NativeEventEmitter**.
+
+```JavaScript
+componentDidMount() {
+  Platform.currentTheme.addListener('themechanged', this.onAppThemeChanged);
```

Comment:
Is 'themechanged' correct or should it be 'appThemeChanged' (not sure if this is a set value or just what's in the AppTheme example in RNTester) ?